crates/partition-store/src/outbox_table/mod.rs

```diff
@@ -18,7 +18,7 @@ use restate_storage_api::{Result, StorageError};
 use restate_types::identifiers::PartitionId;
 use restate_types::storage::StorageCodec;
 use std::io::Cursor;
-use std::ops::Range;
+use std::ops::RangeInclusive;
```

Comment:
I changed this without too much thought since it more accurately captures the intent behind the truncate operation but I only afterwards saw that unlike the regular `Range`, the `start` and `end` fields aren't public, which requires a method call. I'm guessing it was a deliberate choice to use `Range`?